UPGRADE.md

```diff
@@ -16,9 +16,12 @@ $ make migrate
 
 ## Unreleased
 
+- If you reused any React components in Django templates through `richie-react`, you should know that the `data-locale` attribute on them is no longer used. Instead, they rely on `<html lang>` to pick up their current locale, which should already be set in all documents anyway.
+- As a result, the `lang` attribute on `<html>`, which is a requirement for an accessible page anyway, is now an absolute requirement. Users who overrode the base template and removed this need to add it again.
+
```

Comment:
One more thing : users who overrode the `REACT_LOCALES` setting (for example to change priority order) should rename the setting to `RFC_5646_LOCALES` and replace their locales by FRC5646 locales in this setting value.